idl/power.idol

```diff
@@ -18,5 +18,35 @@ Interface(
             ),
             idempotent: true,
         ),
+        "bmr491_event_log_read": (
+            doc: "reads an event from the BMR491's combined fault and lifecycle event log",
+            encoding: Hubpack,
```

Comment:
Do we need `encoding: Hubpack` on these new operations? It looks like their reply types all work with `zerocopy`.

Reply:
Good idea, done